UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -55,16 +52,27 @@ public override void ImplantPeriodicUpdate()
 			DoHeartBeat();
 		}
 
+		public override void RemovedFromBody(LivingHealthMasterBase livingHealth)
+		{
+			livingHealth.CirculatorySystem.Hearts.Remove(this);
+		}
+
+		public override void AddedToBody(LivingHealthMasterBase livingHealth)
```

Comment:
this function's name (as well as `RemovedFromBody()` make it sound like it's a bool rather than a void, `AddToBody()` would be a clearer name though this is just a nitpick.